src/test/scala/is/hail/utils/IntervalSuite.scala

```diff
@@ -150,8 +167,15 @@ class IntervalSuite extends TestNGSuite {
   }
 }
 
+object SetInterval {
+  def from(i: Interval): SetInterval =
+    SetInterval(i.start.asInstanceOf[Int], i.end.asInstanceOf[Int], i.includeStart, i.includeEnd)
+}
+
 case class SetInterval(start: Int, end: Int, includeStart: Boolean, includeEnd: Boolean) {
 
+  val pord: ExtendedOrdering = TInt32().ordering
+
```

Comment:
Clever. Suggest renaming `doublepointset` to `doubledPointSet`(it's a set of integers, not doubles) and making it private

Reply:
I can't make it private without rewriting the IntervalTree construction.